packages/app/package.json

```diff
@@ -9,7 +9,8 @@
       "@types/react": "17.0.37",
       "@types/react-dom": "17.0.11",
       "@ifixit/tsconfig": "workspace:*",
-      "typescript": "4.5.3"
+      "typescript": "4.5.3",
+      "react": "16.14.0 || 17.0.2"
```

Comment:
I think we only use `react 17.0.2` in this project. Where is `16.14.0` coming from, is that what we use in `iFixit/ifixit`?

Reply:
Yes, there are many places in `iFixit/ifixit` that use `16.14.0`. `16.14.0` is the preferred version on `iFixit/ifixit`, actually